docs/sources/enterprise/saml.md

```diff
@@ -204,6 +204,61 @@ org_mapping = Engineering:2, Sales:3
 allowed_organizations = Engineering, Sales
 ```
 
+## Set up SAML with Okta
+
+To configure SAML integration with Okta, create integration inside the Okta org first.
+
+1. Log in to the [Okta portal](https://login.okta.com/).
+1. Go to the Admin Console in your Okta org by clicking **Admin** in the upper-right corner. If you are in the Developer Console, click **Developer Console** in the upper-left corner and then click **Classic UI** to switch over to the Admin Console.
+1. In the Admin Console, go to **Applications** > **Applications**.
+1. Click **Add Application**.
+1. Click **Create New App** to start the Application Integration Wizard.
+1. Choose **Web** as a platform.
+1. Select **SAML 2.0** in the Sign on method section.
+1. Click **Create**.
+1. On the **General Settings** tab, enter a name for your Grafana integration and optionally upload a logo.
```

Comment:
```suggestion
1. On the **General Settings** tab, enter a name for your Grafana integration. You can also upload a logo.
```

Reply:
What is the logo used for? Is it for the Grafana login page?

Reply:
Logo is used on the Okta main page and in the application lists.

![Screenshot from 2020-05-14 08-26-27](https://user-images.githubusercontent.com/4932851/81896160-a1d98980-95bc-11ea-9148-9c00ccbe5daf.png)